README.md

```diff
@@ -191,7 +191,7 @@ If you're using CocoaPods, add this to your `Podfile` adjusting the `:tag` to
 match the `[tag_name]` you used to build the plugin above:
 
 ```ruby
-pod 'SwiftProtobuf', '~> 1.0.1'
+pod 'SwiftProtobuf', '~> 1.0.0'
```

Comment:
For both of these we can do ~> 1.0 if we're sure 1.1, 1.2, etc. are always gonna be api compatible.

Reply:
I certainly plan to enforce API compatibility within a major version.

Reply:
Updated to just be `~> 1.0` then.